code/modules/jobs/jobs.dm

```diff
@@ -59,6 +59,24 @@ GLOBAL_LIST_INIT(nonhuman_positions, list(
 	"Cyborg",
 	"pAI"))
 
+GLOBAL_LIST_INIT(exp_jobsmap, list(
+	EXP_TYPE_CREW = list(titles = command_positions | engineering_positions | medical_positions | science_positions | supply_positions | security_positions | civilian_positions | list("AI","Cyborg")), // crew positions
+	EXP_TYPE_COMMAND = list(titles = command_positions),

```

Comment:
`titles` should be in quotes. Byond allows you to not (for lists), but now if somebody creates a global var called titles byond could decide to replace this text with that.